source/pstate/pstate.go

```diff
@@ -24,6 +24,7 @@ import (
 // Source implements FeatureSource.
 type Source struct{}
 
+// Name returns a friendly name for this source.
```

Comment:
Maybe change this description to `Name returns a identifier string for this feature source`?